app/models/concerns/inventory_computer.rb

```diff
@@ -17,6 +19,10 @@ def historical_quantity
         SumAsOf.new(PackagesInventory)
       end
 
+      def storage_type
+        SumAsOf.new(StorageType)
```

Comment:
`StorageType` cannot be summed, so this doesn't make sense 🤔

Reply:
yeah. corrected it as mentioned in above comment